packages/practice/checkin/methods-server.coffee

```diff
@@ -1,7 +1,20 @@
 LOI = LandsOfIllusions
 PAA = PixelArtAcademy
+Twit = Npm.require 'twit'
 
 Meteor.methods
+  getImgFromTweet: (id) ->
```

Comment:
All methods should haves `check`s for all parameters. Look into [this](http://docs.meteor.com/#/full/check) and we also use [these](https://github.com/peerlibrary/meteor-check-extension) custom ones.